app/scenes/Trash/index.tsx

```diff
@@ -27,7 +27,7 @@ function Trash() {
             action={permanentlyDeleteDocumentsInTrash}
             context={context}
           >
-            Empty
+            {t("Empty trash")}
```

Comment:
```suggestion
            {t("Empty")}
```

This one was purposeful, "trash" is assumed as it's the screen being viewed